src/extensions/default/Inline3DParametersEditor/AframeParameters.json

```diff
@@ -0,0 +1,6 @@
+{     
```

Comment:
We haven't really talked about "AFrame" in this code up until now, and these are pretty generic "3D" parameters.  Should we just call this `Parameters.json`?